src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
```

Comment:
Now that we have the Mount Service we should in most cases only use that in open DC/OS. Because `hasCapabilities` is an enterprise feature and not available in open DC/OS I think it shouldn't live here.

The list of Service create options should be wrapped in a Mount Component and there should be a plugin that listens to that Mount Component and adds an extra option, in this case - Universe, if the user `hasCapability`. Does that make sense or am I missing something? :)